src/style.css

```diff
@@ -1,3 +1,197 @@
+
+  padding-top: 32px;
+  padding-left: 19px;
+}
+
+img {
+  width: 100%;
+  height: 100%;
+}
+
+.card__text {
+  width: 166px;
+  height: auto;
+
+  margin-top: 40px;
+  margin-left: 16px;
+  
+  color: var(--Main-accent, #060b35);
+  font-size: 12px;
+  font-style: normal;
+  font-weight: 500;
+  line-height: 18px; /* 150% */
+}
+
+.card__productCode {
+  width: 100px;
+  height: auto;
+
+  margin-top: 4px;
+  margin-left: 16px;
+
+  color: var(--Secondary, #616070);
+  font-size: 10px;
+  font-style: normal;
+  font-weight: 400;
+  line-height: 14px; /* 140% */
+}
+
+.card__reviews {
+  display: flex;
+  justify-content: space-between;
+  align-items: flex-end;
+
+  width: 166px;
+  height: auto;
+
+  margin-top: 16px;
+  margin-left: 16px;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  display: flex;
+  text-align: center;
+  justify-content: center;
+  margin-right: 4px;
+  width: 16px;
+  height: 16px;
+  background-image: url(images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars__star:last-child {
+  margin-right: 0;
+}
+
+.stars--5 .stars__star,
+.stars--1 .stars__star:nth-child(-n+1),
+.stars--2 .stars__star:nth-child(-n+2),
+.stars--3 .stars__star:nth-child(-n+3),
+.stars--4 .stars__star:nth-child(-n+4) {
+  background-image: url(images/star-active.svg);
+}
+
+.card__reviewNumber {
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-end;
+  flex-shrink: 0;
+
+  width: 53px;
+  height: 16px;
+
+  color: var(--Main-accent, #060b35);
+  font-size: 10px;
+  font-style: normal;
+  text-align: right;
+  font-weight: 400;
+  line-height: 14px; /* 140% */
+  margin: 0;
+}
+
+.card__price {
+  display: flex;
+  justify-content: space-between;
+
+  width: 166px;
+  height: 18px;
+
+  margin-left: 16px;
+  margin-top: 24px;
+}
+
+.card__priceText {
+  color: var(--Secondary, #616070);
+  font-size: 12px;
+  font-style: normal;
+  font-weight: 400;
+  line-height: 18px; /* 150% */
+  margin: 0;
+}
+
+.card__priceNumber {
+  width: 50px;
+  
+  color: var(--Main-accent, #060b35);
+  text-align: center;
+  font-size: 16px;
+  font-style: normal;
+  font-weight: 700;
+  line-height: 18px; /* 112.5% */
+}
+
+.card__button {
+  width: 166px;
+  height: 40px;
+  flex-shrink: 0;
+
+  margin-left: 16px;
+  margin-top: 16px;
+  margin-bottom: 16px;
+  padding: 0;
+
+  border-radius: 5px;
+  background: var(--Blue-accent, #00acdc);
+
+  border: none;
+  cursor: pointer;
+
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.card__buttonText {
+  height: 16px;
+  width: 27px;
+
+  color: var(--White, #fff);
+  text-align: center;
+  font-size: 13px;
+  font-style: normal;
+  font-weight: 700;
+  line-height: 16px; /* 114.286% */
+  text-transform: uppercase;
+}
+
+.card__buttonText:hover {
+  color: var(--Blue-accent, #00acdc);
+  margin: 0;
+  padding: 0;
+}
+
+.card__button:hover {
+  background-color: #fff; /* Змініть колір фону при наведенні курсору */
```

Comment:
<img width="367" alt="Screenshot 2023-10-10 at 14 48 33" src="https://github.com/mate-academy/layout_product-cards/assets/135232001/e26d6456-7b58-4bb8-b5cb-f3659bb1fa6f">
The text color should be applied on button hover to avoid text disappearing